config/locales/client.en.yml

```
@@ -4,6 +4,10 @@ en:
       site_settings:
         categories:
           discourse_category_experts: "Discourse Category Experts"
+      web_hooks:
+        post_event:
+          category_experts_approved: "Post marked as category experts post"
+          category_experts_unapproved: "Post unmarked as category experts post"
```

```suggestion
          category_experts_approved: "Post is marked as category experts post"
          category_experts_unapproved: "Post is unmarked as category experts post"
```

to stick to the pattern